Libraries/LibWeb/CSS/Interpolation.cpp

```diff
@@ -1559,4 +1559,120 @@ RefPtr<StyleValue const> interpolate_value(DOM::Element& element, CalculationCon
     return interpolate_discrete(from, to, delta, allow_discrete);
 }
 
+template<typename T>
+static T composite_raw_values(T underlying_raw_value, T animated_raw_value)
+{
+    return underlying_raw_value + animated_raw_value;
+}
+
+RefPtr<StyleValue const> composite_value(StyleValue const& a_underlying_value, StyleValue const& a_animated_value, Bindings::CompositeOperation composite_operation)
```

Comment:
Nit: I'm not a fan of the `a_` prefix of these, and I don't think `underlying_value` and `animated_value` would collide with anything.

Reply:
I've renamed the variables that were causing collisions.